backend/src/org/commcare/suite/model/graph/BubbleSeries.java

```diff
@@ -18,6 +18,13 @@
 public class BubbleSeries extends XYSeries {
     private String mRadius;
     private XPathExpression mRadiusParse;
+    
+    /*
+     * Deserialization Only!
```

Comment:
@wpride For Sure!

The deserializtion engine has to be able to inflate a copy of each class using reflection, then it asks that class to deserialize itself against the current position in the input stream, as such it needs to have a constructor with no arguments to be able to instantiate a new copy.

In a world with no 1.3 compatibility, we'd likely want these constructors to be private and then we'd change their permissions through reflection so that the API was more clear, but so it goes.